src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -418,6 +430,25 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
                                custom_locations_oid, helm_client_location, enable_private_link, arm_metadata, onboarding_timeout, container_log_path)
     return put_cc_response
 
+def validate_existing_provisioned_cluster_for_reput(cluster_resource, kubernetes_distro, kubernetes_infra, enable_private_link, private_link_scope_resource_id, distribution_version, azure_hybrid_benefit, location):
+    if cluster_resource.kind == "ProvisionedCluster":
+        if azure_hybrid_benefit is not None:
+            raise InvalidArgumentValueError("Updating the 'azure hybrid benefit' property of a Provisioned Cluster is not supported from the Connected Cluster CLI. Please use the 'az aksarc update' CLI command.\nhttps://learn.microsoft.com/en-us/cli/azure/aksarc?view=azure-cli-latest#az-aksarc-update")
```

Comment:
>Please use the 'az aksarc update' CLI command [](http://example.com/codeflow?start=164&length=45)

Why not leave this out as well?  The link can explain this too.  And, any time there is an update, it will be only in the documentation - the code need not change.  Discuss with the PM too and see if this makes sense.

Reply:
It was the guidance from a PM to have this error message separately listed - I shared a design document with you in a separate thread for your reference.